statistics/histogram.go

```diff
@@ -1217,6 +1245,7 @@ func (c *Column) GetColumnRowCount(sctx sessionctx.Context, ranges []*ranger.Ran
 				return 0, errors.Trace(err)
 			}
 			cnt -= lowCnt
+			cnt = clampRowCount(cnt, c.notNullCount())
```

Comment:
Why should we add this? It seems to recalculate in line1261.

Reply:
Need to prevent `cnt` from becoming negative here.